src/harness/sas.py

```diff
@@ -76,6 +75,40 @@ def _Request(url, request, ssl_cert, ssl_key):
   logging.debug('Response:\n' + body)
   return json.loads(body)
 
+def _RequestGet(url, ssl_cert, ssl_key):
```

Comment:
I think this implementation is actually POST, not GET. All of the SAS-CBSD and data injection RPCs are HTTP POST.